single-node-k8s.sh

```diff
@@ -100,18 +102,7 @@ function install_minikube()
     # copy kubectl from minikube to /usr/bin
     cp ~/.minikube/cache/linux/"${k8s_version}"/kubectl /usr/bin/
 
-    # add disks to the minikube VM
```

Comment:
```
+ /usr/local/bin/minikube start --force --memory=4096 --cpus=12 -b kubeadm --kubernetes-version=v1.22.0 --driver=kvm2 --feature-gates=ExpandCSIVolumes=true --cni=bridge --extra-config=kubelet.resolv-conf=/run/systemd/resolve/resolv.conf --extra-config=kubelet.read-only-port=10255 --extra-config=apiserver.enable-admission-plugins=PodSecurityPolicy --wait-timeout=10m --wait=all ' --extra-disks=3 --disk-size=32g '
😄  minikube v1.23.0 on Fedora 34
    ▪ KUBECONFIG=/home/mrajanna/workspace/src/github.com/JohnStrunk/k8s-cluster/kubeconfig
    ▪ MINIKUBE_VERSION=v1.23.0
❗  minikube skips various validations when --force is supplied; this may lead to unexpected behavior
🆕  Kubernetes 1.22.1 is now available. If you would like to upgrade, specify: --kubernetes-version=v1.22.1
✨  Using the kvm2 driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🔄  Restarting existing kvm2 VM for "minikube" ...
🐳  Preparing Kubernetes v1.22.0 on Docker 20.10.8 ...
    ▪ kubelet.resolv-conf=/run/systemd/resolve/resolv.conf
    ▪ kubelet.read-only-port=10255
    ▪ apiserver.enable-admission-plugins=PodSecurityPolicy
🔗  Configuring bridge CNI (Container Networking Interface) ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: storage-provisioner, default-storageclass
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+ [[ kvm2 != \n\o\n\e ]]
+ wait_for_ssh
+ local tries=100
+ (( tries > 0 ))
+ /usr/local/bin/minikube ssh echo connected
+ return 0
+ /usr/local/bin/minikube ssh 'sudo mkdir -p /mnt/vda1/var/lib/rook;sudo ln -s /mnt/vda1/var/lib/rook /var/lib/rook'
+ /usr/local/bin/minikube kubectl -- cluster-info
Kubernetes control plane is running at https://192.168.39.75:8443
CoreDNS is running at https://192.168.39.75:8443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy

To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
[🎩︎]mrajanna@fedora ceph-csi $]minikube ssh 'echo 1 | sudo tee /sys/bus/pci/rescan > /dev/null ; dmesg | grep virtio_blk'


[    2.924530] virtio_blk virtio2: [vda] 40960000 512-byte logical blocks (21.0 GB/19.5 GiB)
[🎩︎]mrajanna@fedora ceph-csi $]minikube ssh

```
@Rakshith-R am not getting extra disk if I run minikube.sh up can you please check on your end you have 3 extra disks on minikube VM?

Reply:
yes, the double quote around `$DISK_CONFIG` are causing the problem :facepalm: , it works properly without them.
PTAL #2498